src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -62,7 +62,9 @@ export default class BusinessLookupFetch extends Mixins(CommonMixin, SearchMixin
         if (this.isConversion) {
           return this.isAlterable || 'This business cannot alter their business type'
         } else if (this.isRestoration) {
-          return this.isRestorable || 'This business cannot be restored.'
+          return this.isRestorable || 'This business cannot be restored'
+        } else if (this.isChangeName) {
+          return this.isNameChangeable || 'This business cannot change their business name'
```

Comment:
There is an issue preventing these validation rules from being shown, I am having a look fixing it.

Reply:
The problem is being handled by [entity/#17592](https://github.com/bcgov/entity/issues/17952). For this ticket, this PR should fix the original bug where users are able to go through the name change flow with PA or PAR entity types. Testing will have to be done while authenticated until 17592 is resolved.

Does anyone know any accounts that have PA or PAR entity types that can be used?

Reply:
PA or PAR entity doesn't have to be associated with an account -- just needs to be searchable in LEAR or COLIN. Ask Riyaz...?

Or, hack the code temporary to fake a found entity's type...?

Reply:
Ah I see, the code Riyaz provided on this ticket (PA0000004) seems to only work in the test environment. I have hacked the code to mimic the issue locally, but will ask if there are any codes that can work for dev.